extensions/arc/src/localizedConstants.ts

```diff
@@ -114,9 +121,33 @@ export const databaseName = localize('arc.databaseName', "Database name");
 export const enterNewPassword = localize('arc.enterNewPassword', "Enter a new password");
 export const confirmNewPassword = localize('arc.confirmNewPassword', "Confirm the new password");
 export const learnAboutPostgresClients = localize('arc.learnAboutPostgresClients', "Learn more about Azure PostgreSQL Hyperscale client interfaces");
+export const scalingCompute = localize('arc.scalingCompute', "scaling compute vCores and memory.");
+export const computeAndStorageDescriptionPartOne = localize('arc.computeAndStorageDescriptionPartOne', "You can scale your Azure Arc enabled");
+export const computeAndStorageDescriptionPartTwo = localize('arc.computeAndStorageDescriptionPartTwo', "PostgreSQL Hyperscale server group by");
+export const computeAndStorageDescriptionPartThree = localize('arc.computeAndStorageDescriptionPartThree', "without downtime and by");
+export const computeAndStorageDescriptionPartFour = localize('arc.computeAndStorageDescriptionPartFour', "Before doing so, you need to ensure");
+export const computeAndStorageDescriptionPartFive = localize('arc.computeAndStorageDescriptionPartFive', "there is sufficient resources available");
+export const computeAndStorageDescriptionPartSix = localize('arc.computeAndStorageDescriptionPartSix', "in your Kubernetes cluster to honor this configuration.");
 export const node = localize('arc.node', "node");
 export const nodes = localize('arc.nodes', "nodes");
+export const workerNodes = localize('arc.workerNodes', "Worker Nodes");
+export const coordinatorNode = localize('coordinatorNode', "Coordinator Node");
 export const storagePerNode = localize('arc.storagePerNode', "storage per node");
+export const workerNodeCount = localize('arc.workerNodeCount', "Worker node count:");
+export const configurationPerNode = localize('arc.configurationPerNode', "Configuration (per node)");
+export const vCoresMax = localize('arc.vCoresMax', "vCores - max:");
+export const vCoresMin = localize('arc.vCoresMin', "vCores - min:");
+export const memoryMax = localize('arc.memoryMax', "Memory - max (in GiB):");
+export const memoryMin = localize('arc.memoryMin', "Memory - min (in GiB):");
+export const dataStorage = localize('arc.dataStorage', "Storage Data Size (in GiB):");
+export const logsStorage = localize('arc.logsStorage', "Storage Logs Size (in GiB):");
+export const backupsStorage = localize('arc.backupsStorage', "Storage Backups Size (in GiB):");
+export const enableAdvancedConfiguration = localize('arc.enableAdvancedConfiguration', "Enable advanced configuration");
+export const workerValidationErrorMessage = localize('arc.workerValidationErrorMessage', "The number of workers cannot be decreased.");
+export const vCoresValidationErrorMessage = localize('arc.vCoresValidationErrorMessage', "Valid Cpu resource quantities are strictly positive and request is greater than 0.");
```

Comment:
I think this validation message is confusing.  The message mentions 'request' even though it's used for validating both `--cores-request` and `--cores-limit`.  Also 'positive' and 'greater than 0' mean the same thing. I'd have 2 messages like:

`CPU request must be positive`
`CPU limit must be positive`